### PR TITLE
Document process to create quantum workspaces with a list of providers.

### PR DESCRIPTION
**Pull request by ricardo-espinoza**
Update on quantum docs to instruct users on the commands required to create quantum workspaces with a list of providers.

### CODE REVIEW

File: articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md [hunk 1]
```diff
@@ -60,8 +60,41 @@ And then you can create it using the following command, using the previous examp
...
 
-> [!NOTE]
-> Adding or removing providers from a workspace using the Az CLI is not currenly supported. For now, you'll need to [use the Azure portal](xref:microsoft.quantum.workspaces-portal) to modify a workspace.
+By default, a new workspace will be created only containing the Microsoft Basic QIO provider. At any time, it is possible to modify a workspace in order to add or remove other providers. For this, you can use [the Azure portal](xref:microsoft.quantum.workspaces-portal) and edit the workspace.
+
+Alternatively, you can create a workspace in a more advanced way and specify the providers using the CLI directly.
+
+## Specify additional providers during creation of an Azure Quantum workspace
+
+1. To retrieve the list of quantum providers available, you can use the following command, using **westus** as example location:
```
Comment (geduardo) [inline articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:69]:
```suggestion
1. To retrieve the list of quantum providers available, you can use the following command (using **westus** as example location) :
```

File: articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md [hunk 2]
```diff
@@ -60,8 +60,41 @@ And then you can create it using the following command, using the previous examp
...
+
+   ```dotnetcli
+   az quantum offerings list -l westus
+   ```
+
+1. You can see a summary table of the information above by specifying the output format:
+
+   ```dotnetcli
+   az quantum offerings list -l westus -o table
```
Comment (geduardo) [inline articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:78]:
```suggestion
   az quantum offerings list -l westus -o table
```

Comment (geduardo) [reply articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:78]: We can skip that step.

File: articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md [hunk 3]
```diff
@@ -60,8 +60,41 @@ And then you can create it using the following command, using the previous examp
...
+   ```
+
+1. Once you determine the provider and SKU to include in your workspace, you can review terms using this command, assuming **MyProviderID** and **MySKU** as example values:
```
Comment (geduardo) [inline articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:81]: What is an SKU?

Comment (ricardo-espinoza) [reply articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:75]: @geduardo , each provider offers multiple plans or SKUs (e.g. "Basic", "Premium", "Pay as you go") and each of them has associated terms/conditions, cost, etc.

**Review by geduardo — APPROVED** [review]

**Pull request merged**